Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -4135,7 +4135,7 @@ NeedsKeyboardEventDisambiguationQuirks:
 
 NeedsSiteSpecificQuirks:
   type: bool
-  status: developer
+  status: embedder
```

Comment:
Isn't this something exposed to Web developers via Web Inspector? Does this qualify as an embedder feature?

Reply:
Right. Since WebInspector already exposes it, we don't need to generate a 'developer' switch for it. I should not have marked it as 'developer', since WebInspector already provides a real UI for it.